Sources/TuistGenerator/Linter/TargetLinter.swift

```diff
@@ -105,19 +105,21 @@ class TargetLinter: TargetLinting {
     private func lintCopiedFiles(target: Target) -> [LintingIssue] {
         var issues: [LintingIssue] = []
 
-        let files = target.resources.map(\.path)
-        let entitlements = files.filter { $0.pathString.contains(".entitlements") }
-
-        if let targetInfoPlistPath = target.infoPlist?.path, files.contains(targetInfoPlistPath) {
-            let reason = "Info.plist at path \(targetInfoPlistPath) being copied into the target \(target.name) product."
-            issues.append(LintingIssue(reason: reason, severity: .warning))
+        let infoPlists = target.settings?.configurations.infoPlists().compactMap(\.path)
```

Comment:
I would add also `target.infoPlist?.path` hear so the if below becomes simpler

Reply:
nice idea, fixed